src/Preflight/Preflight.php

```diff
@@ -199,6 +200,15 @@ protected function doRun($argv)
         $runner = new \Robo\Runner();
         $runner->registerCommandClasses($application, $commandClasses);
 
+        // Process legacy Drush global options.
+        if ($input->getParameterOption(['--yes', '-y'], false, true)) {
+            $input->setInteractive(false);
+        }
+        // Use -vvv for even more verbose logging.
+        if ($input->getParameterOption(['--debug', '-d'], false, true)) {
+            $output->setVerbosity(Output::VERBOSITY_VERBOSE);
```

Comment:
VERBOSITY_VERBOSE is `-v`. VERBOSITY_VERY_VERBOSE is `-vv`. `VERBOSITY_DEBUG` is `-vvv`. I think we should use `VERBOSITY_DEBUG` here.